global/build/global.css

```diff
@@ -4927,6 +5040,15 @@ Single image/video component
   --tw-gradient-to: rgb(229 8 8 / 0) var(--tw-gradient-to-position);
   --tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to);
 }
+.su-from-white{
+  --tw-gradient-from: #fff var(--tw-gradient-from-position);
+  --tw-gradient-to: rgb(255 255 255 / 0) var(--tw-gradient-to-position);
+  --tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to);
+}
+.su-via-\[rgb\(255_255_255\/\.5\)_8\%\]{
+  --tw-gradient-to: rgb(255 255 255 / 0)  var(--tw-gradient-to-position);
+  --tw-gradient-stops: var(--tw-gradient-from), rgb(255 255 255/.5) 8% var(--tw-gradient-via-position), var(--tw-gradient-to);
+}
```

Comment:
Oooooh. Thanks for this. I think your recommendation will work.

I got my janky shorthand here https://github.com/tailwindlabs/tailwindcss/discussions/10705 because I was looking for a way to use rgba values in a gradient and the Tailwind Cheat Sheet didn't have this info.

Reply:
Oh yeah, I think the Tailwind cheat sheet might be a bit outdated. I haven't looked at it for a while, but TW keeps coming out with new features 😄